setup.cfg

```
@@ -37,7 +37,7 @@ project_urls =
 packages = find:
 install_requires =
     lazy_object_proxy>=1.4.0
-    wrapt>=1.11,<1.14
+    wrapt>=1.11,<1.15
```

```suggestion
    wrapt>=1.11,<2
```
I think it's fairly safe to do.